.github/workflows/nightly-ubuntu.yml

```diff
@@ -0,0 +1,447 @@
+name: Nightly-Ubuntu
+
+# This is a manual trigger and schedule to run everyday at 2AM CST.
+on:
+  workflow_dispatch:
+  schedule:
+    # Schedule to run everyday at 2AM CST
+    - cron:  '0 18 * * *'
+
+env:
+    WORK_DIR: /root/pkgs
+    HOME: /root
+    OCCLUM_VERSION: 0.18.1
+    kubernetes_version: 1.16.9
+    nap_time: 60
+
+jobs:
+  k8s_ubuntu:
+    runs-on: [self-hosted, ubuntu]
```

Comment:
Change label to [self-hosted, nightly]
I will add this machine labeled "self-hosted, nightly" to show that this machine is specially designed for nightly testing.